src/utils/redux-helpers/selector-helpers.js

```diff
@@ -0,0 +1,43 @@
+const _ = require('lodash');
```

Comment:
This just came with the other redux helpers, right?

Reply:
Yes lodash is only visiting us because it's heavily baked-into the helpers

Reply:
Sorry, I meant this file